src/libraries/System.Formats.Tar/tests/TarTestsBase.cs

```diff
@@ -613,12 +613,27 @@ public static IEnumerable<object[]> GetPaxAndGnuTestCaseNames()
             }
         }
 
-        public static void ExtractWithTarTool(string source, string destination)
+        public static bool CanExtractWithTarTool = CanExtractWithTarTool_Method();
+
+        private static bool CanExtractWithTarTool_Method()
         {
-            Assert.True(!OperatingSystem.IsWindows(), "This method is unsupported on Windows.");
+            using Process tarToolProcess = new Process();
+            tarToolProcess.StartInfo.FileName = OperatingSystem.IsWindows() ? "tar" : "/usr/bin/tar";
```

Comment:
As noted above, if this works, it would presumably enable the test on Alpine (do we still run PR validation on Alpine)?
```suggestion
            tarToolProcess.StartInfo.FileName = "tar"; // location varies: find it on the PATH
```

Reply:
> do we still run PR validation on Alpine

Yes, Alpine 3.14 was one of the failing legs of the previous commit, along with:
Debian 11
Ubuntu-18.04-bionic
Debian 10